src/main/java/net/minecraftforge/event/entity/living/LivingChangeTargetEvent.java

```diff
@@ -0,0 +1,74 @@
+package net.minecraftforge.event.entity.living;
+
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.living.LivingSetAttackTargetEvent.ILivingTargetType;
+import net.minecraftforge.eventbus.api.Event.HasResult;
+
+/**
+ * This event allows you to change the target an entity has. <br>
+ * This event is fired before {@link LivingSetAttackTargetEvent}. <br>
+ * <br>
+ * This event is fired via the {@link ForgeHooks#}
```

Comment:
Not finished javadoc.

Reply:
Fixed: c8274622fe30303ec1028dfd70d3841605bb9459